ui/src/components/issuers.tsx/IssuersTable.tsx

```diff
@@ -0,0 +1,193 @@
+    isAsyncTaskDataAvailable(issuersList) ? issuersList.data : []
+  );
+  const [searchParams, setSearchParams] = useSearchParams();
+
+  const queryParam = searchParams.get(QUERY_SEARCH_PARAM);
+
+  const handleIssuerChange = ({ target: { value } }: RadioChangeEvent) => {
+    const parsedIdentifier = identifierParser.safeParse(value);
+
+    if (parsedIdentifier.success) {
+      handleChange(parsedIdentifier.data);
+    } else {
+      handleChange(null);
+    }
+  };
+
+  useEffect(() => {
+    if (isAsyncTaskDataAvailable(issuersList)) {
+      if (!queryParam) {
+        setFilteredIdentifiers(issuersList.data);
+      } else {
+        const filteredData = issuersList.data.filter((issuer: Issuer) =>
+          Object.values(issuer).some((value: string) => value.includes(queryParam))
+        );
+        setFilteredIdentifiers(filteredData);
+      }
+    }
+  }, [queryParam, issuersList]);
+
+  const onSearch = useCallback(
+    (query: string) => {
+      setSearchParams((previousParams) => {
+        const previousQuery = previousParams.get(QUERY_SEARCH_PARAM);
+        const params = new URLSearchParams(previousParams);
+
+        if (query === "") {
+          params.delete(QUERY_SEARCH_PARAM);
+        } else if (previousQuery !== query) {
+          params.set(QUERY_SEARCH_PARAM, query);
+        }
+
+        return params;
+      });
+    },
+    [setSearchParams]
+  );
+
+  const tableColumns: TableColumnsType<Issuer> = [
+    {
+      align: "center",
+      dataIndex: "identifier",
+      key: "identifier",
+      render: (identifier: Issuer["identifier"]) => <Radio.Button value={identifier} />,
+    },
+    {
+      dataIndex: "identifier",
+      key: "identifier",
+      render: (identifier: Issuer["identifier"]) => (
+        <Typography.Text strong>{identifier}</Typography.Text>
+      ),
+      sorter: ({ identifier: a }, { identifier: b }) => a.localeCompare(b),
+      title: "Did",
+    },
+    {
+      dataIndex: "authBJJCredentialStatus",
+      key: "authBJJCredentialStatus",
+      render: (credentialStatus: Issuer["authBJJCredentialStatus"]) => (
+        <Typography.Text strong>{credentialStatus}</Typography.Text>
+      ),
+      sorter: ({ authBJJCredentialStatus: a }, { authBJJCredentialStatus: b }) =>
+        a.localeCompare(b),
+
+      title: "Credential Status",
+    },
+    {
+      dataIndex: "network",
+      key: "network",
+      render: (network: Issuer["network"], { blockchain }: Issuer) => (
+        <Typography.Text strong>
+          {blockchain}-{network}
+        </Typography.Text>
+      ),
+      sorter: (
+        { blockchain: blockchainA, network: networkA },
+        { blockchain: blockchainB, network: networkB }
+      ) => `${blockchainA}-${networkA}`.localeCompare(`${blockchainB}-${networkB}`),
+      title: "Network",
+    },
+  ];
+
+  const addButton = (
+    <Button icon={<IconPlus />} onClick={handleAddIssuer} type="primary">
+      Add new issuer
```

Comment:
I'm not especially happy having all the literals in constants, but this is something from the past that we should honor until we discard, so this should be `ISSUER_ADD`